### PR TITLE
added changes from 1855

### PR DESCRIPTION
**Pull request by cpholguera**
to be merged instead of #1855 due to inactivity and different location of content.

### CODE REVIEW

File: Document/0x08-Testing-Tools.md [hunk 1]
```diff
@@ -68,8 +68,8 @@ Independently of the chosen mode, you can make use of the [Frida JavaScript APIs
 Frida also provides a couple of simple tools built on top of the Frida API and available right from your terminal after installing frida-tools via pip. For instance:
 
 - You can use the [Frida CLI](https://www.frida.re/docs/frida-cli/ "Frida CLI") (`frida`) for quick script prototyping and try/error scenarios.
-- [`frida-ps`](https://www.frida.re/docs/frida-ps/ "frida-ps") to obtain a list of all apps (or processes) running on the device including their names and PDIs.
-- [`frida-ls-devices`](https://www.frida.re/docs/frida-ls-devices/ "frida-ls-devices") to list your connected devices;
+- [`frida-ps`](https://www.frida.re/docs/frida-ps/ "frida-ps") to obtain a list of all apps (or processes) running on the device including their names, identifiers and PDIs.
```
Comment (sushi2k) [inline Document/0x08-Testing-Tools.md:71]:
```suggestion
- [`frida-ps`](https://www.frida.re/docs/frida-ps/ "frida-ps") to obtain a list of all apps (or processes) running on the device including their names, identifiers and PIDs.
```

**Review by sushi2k — CHANGES_REQUESTED** [review]
One tiny change :-)

**Comment by cpholguera** [comment]
Thanks @sushi2k, done!

**Review by sushi2k — APPROVED** [review]
LGTM

**Pull request merged**